tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -23,7 +23,6 @@
 from ..executors.executormarionette import (MarionetteTestharnessExecutor,
                                             MarionetteRefTestExecutor,
                                             MarionetteWdspecExecutor)
-from ..environment import hostnames
```

Comment:
This is also used later in the file, right?

Reply:
Apparently so. But it passed CI despite that because we disable the unused variable pyflake still. :(